crates/runtime/benches/setup/mod.rs

```diff
@@ -70,7 +71,15 @@ fn build_app(
     connector: &str,
     acceleration: Option<&Acceleration>,
 ) -> App {
-    let mut app_builder = AppBuilder::new("runtime_benchmark_test");
+    // Disable result caching in benchmark test
+    let results_cache = ResultsCache {
```

Comment:
Like it. Just a note that we currently run queries directly using datafusion.sql functionality instead of `query` api so results cache is not used. If we can update this as well it will be great.

Reply:
Got it, will remove this update